wls-exporter-core/src/main/java/com/oracle/wls/exporter/WebClient8Impl.java

```diff
@@ -0,0 +1,193 @@
+// Copyright 2020 Oracle Corporation and/or its affiliates.
```

Comment:
// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.

Reply:
Missing "(c)" and comma